crowdfunding_compassion/__manifest__.py

```diff
@@ -55,13 +56,11 @@
         "views/product_template_view.xml",
         "views/projects_list_page.xml",
         "views/project_creation_page.xml",
-        "views/crowdfunding_components.xml",
         "views/project_page.xml",
-        "views/progress_bar.xml",
```

Comment:
Why removing this view?

Reply:
It was both in crowdfuning_components and here, to I thought this was an old file. I can split it again, but I don't know if we should group reused components in one file or split them by function ?

Reply:
It's ok then, I thought it was a mistake and didn't know the view is already in another file.